response_operations_ui/views/messages.py

```diff
@@ -240,7 +240,8 @@ def view_selected_survey(selected_survey):
                                page=page,
                                breadcrumbs=breadcrumbs,
                                messages=messages,
-                               selected_survey=formatted_survey,
+                               selected_survey=selected_survey,
+                               formatted_survey=formatted_survey,
```

Comment:
Could we have a more descriptive name ? I am looking at this and wondering why formatted ? Why selected ? Perhaps name_for_display and selected_survey_short_name ( or something better ?)

Reply:
I've changed the formatted name to `displayed_short_name`